browser/ui/toolbar/brave_bookmark_context_menu_controller.cc

```diff
@@ -0,0 +1,102 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/toolbar/brave_bookmark_context_menu_controller.h"
+
+#include <memory>
+#include <vector>
+
+#include "chrome/app/chrome_command_ids.h"
+#include "chrome/browser/ui/bookmarks/bookmark_context_menu_controller.h"
+#include "chrome/grit/generated_resources.h"
+
+BraveBookmarkContextMenuController::BraveBookmarkContextMenuController(
```

Comment:
unit tests? Similar to BookmarkContextMenuControllerTest?

Reply:
oddly it looks like we already have BraveBookmarkContextMenuControllerTest, but it's in a strange place. We should move it here I think

Reply:
done